scipy/stats/_stats_py.py

```diff
@@ -7617,11 +7624,23 @@ def ks_1samp(x, cdf, args=(), alternative='two-sided', method='auto'):
 
     Returns
     -------
-    statistic : float
-        KS test statistic, either D, D+ or D- (depending on the value
-        of 'alternative')
-    pvalue : float
-        One-tailed or two-tailed p-value.
+    res: KstestResult
+        An object containing attributes:
+
+        statistic : float
+            KS test statistic, either D, D+ or D- (depending on the value
+            of 'alternative')
+        pvalue : float
+            One-tailed or two-tailed p-value.
+        statistic_location: float
+            Value of the stochastic variable closest to the KS statistic. By
+            definition, the distance between the theoretical and sampled
+            distributions at this value is largest.
+        statistic_sign: str
+            For alternative == 'two-sided', this attribute is set to '-' if the
```

Comment:
I will probably have a suggested wording for this, too. I don't want to write a concrete suggestion yet, but I'm thinking we'll want to say something like `+1` if the statistic is the maximum _positive_ difference between the ____ CDF and the ____ CDF (can't remember which is theoretical vs empirical) and `-1` if the statistic is the maximum _negative_ difference...